.github/workflows/project-bot.yaml

```diff
@@ -0,0 +1,50 @@
+name: Add a new item to project
+on:
+  issues:
+    types:
+      - opened
+  pull_request:
+    types:
+      - opened
+jobs:
+  add_item:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Issue an access token
+        uses: cybozu/octoken-action@v1
+        id: create-iat
+        with:
+          github_app_id: ${{ secrets.PROJECT_APP_ID }}
+          github_app_private_key: ${{ secrets.PROJECT_APP_PEM }}
+      - name: Get project data
+        env:
+          GITHUB_TOKEN: ${{ steps.create-iat.outputs.token }}
+          ORGANIZATION: topolvm
+          PROJECT_NUMBER: "2"
+        run: |
+          proj_id="$(gh api graphql -f query='
+            query($org: String!, $number: Int!) {
+              organization(login: $org){
+                projectV2(number: $number) {
+                  id
+                }
+              }
+            }' -F org=$ORGANIZATION -F number=$PROJECT_NUMBER --jq '.data.organization.projectNext.id')"
+          echo "PROJECT_ID=${proj_id}" >> $GITHUB_ENV
+      - name: Add an item to project
+        env:
+          GITHUB_TOKEN: ${{ steps.create-iat.outputs.token }}
+        run: |
+          if [ ${{ github.event_name }} = issues ]; then
+            CONTENT_ID=${{ github.event.issue.node_id }}
+          else
+            CONTENT_ID=${{ github.event.pull_request.node_id }}
+          fi
+          item_id="$( gh api graphql -f query='
```

Comment:
`item_id` seems to be unused. How about printing whole response?

Reply:
Nice idea.